site/_posts/2020-12-18-contour_v1110.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Announcing Contour v1.11.0
+excerpt: This blog post covers the new Contour Operator released in v1.11.0.
+author_name: Steve Sloka & Daneyon Hansen
+# author_avatar: /img/contributors/steve-sloka.png
+categories: [kubernetes]
+# Tag should match author to drive author pages
+tags: ['Contour Team']
+---
+
+Contour continues to add new features to help you better manage ingress operations in a cluster.
+Our latest feature release, Contour 1.11.0, now includes support for a new operator aimed at managing the lifecycle of Contour and its corresponding Envoy instances.
+
+## Contour Operator
+
+__FEATURE STATE:__ Contour v1.11.0 [alpha](https://projectcontour.io/resources/deprecation-policy/)
+
+[Contour Operator](https://github.com/projectcontour/contour-operator/blob/main/README.md) provides a method for packaging,
+deploying, and managing Contour. The operator extends the functionality of the Kubernetes API to create, configure, and
+manage instances of Contour on behalf of users.
+It builds upon the basic Kubernetes resource and controller concepts, but includes domain-specific knowledge to automate the entire lifecycle of Contour.
+
+In Kubernetes, controllers of the control-plane implement control loops that repeatedly compare the desired state of the cluster to its actual state.
+If the cluster's actual state doesn’t match the desired state, then the controller takes action to fix the problem.
+Contour Operator is a custom Kubernetes controller that uses the `Contour` custom resource (CR) to manage Contour and its dependent components, i.e. the Envoy DaemonSet.
+The `contours.operator.projectcontour.io` Custom Resource Definition (CRD) defines the `Contour` CR. A `Contour` is handled by the Kubernetes API just like built-in objects, including interaction via kubectl and inclusion in role-based access control (RBAC) policies.
```

Comment:
Should that `contours.operator.projectcontour.io` be `contour.operator.projectcontour.io`? I genuinely don't know, so please correct me if I'm wrong.

Reply:
Nope I think that's right, it shows the plural which is `contours` (yup sounds werid).

Reply:
Ok great, thanks :)